.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # Ignore all files and folders in root
 *
+!/abs.yaml
```

Comment:
```suggestion
!abs.yaml
```

Reply:
Do we want to break with the existing style in the file?

Can we have `abs.yaml` in subdirectories?

Reply:
no all good, better the `!/abs.yaml` you used